tensorly/decomposition/CMTF_ALS

```diff
@@ -0,0 +1,165 @@
+import tensorly as tl
+import tensorly.tenalg
+import tensorly.random
+import numpy as np
+
+
+def rescale_to_unit_frobenius(matrix):
+    # rescale every column of matrix
+    return matrix / tl.norm(matrix, axis=0)
+
+
+def solve_ls(A, B):
+    # solve ||B-AX||^2 (AX = B -> X = A^{-1} @ B)
+    X = tl.pinv(A) @ B
+    return X
+
+
+def calculate_f(X, Y, A, B, C, V):
+    # calculate value of f in cmtf
+    return 1 / 2 * tl.norm(X - tl.kruskal_tensor.kruskal_to_tensor((None, [A, B, C]))) ** 2 + 1 / 2 * tl.norm(
+        Y - A @ V.T)
+
+
+def factor_match_score_3d(A, B, C, V, A_hat, B_hat, C_hat, V_hat):
```

Comment:
Maybe we can use clearer names for the variables?
Also, here, perhaps we can have a signature like:

```suggestion
def factor_match_score_3d(kruskal_tensor_true, kruskal_tensor_pred):
```

and the unroll as :
```
shape, rank = tl.kruskal_tensor._validate_kruskal_tensor(kruskal_tensor)
weights_true, factors_true = kruskal_tensor
...
```